docs/source/working_with_sources.rst

```diff
@@ -56,6 +56,12 @@ To create a network source, use the following steps:
 
    # qpc source add --type network --name mynetwork --hosts /home/user1/hosts_file --cred roothost1 roothost2
 
+ 
+   **TIP:** You can optionally supply an exclude-hosts field that will omit hosts from a scan.
```

Comment:
I would say `For network sources, you can optionally . . .` or you can word it how you want, we just need to specify that this option is only available for network sources.

Reply:
This is already under the 'Creating a Network Source' section.

Reply:
No need! I didn't see that. You can leave it how it is 👍